tutorials/1. Basic PDE solving.ipynb

```diff
@@ -0,0 +1,1231 @@
+{
```

Comment:
<p>The title repeats the previous one. Can be changed to something like</p><p><strong>Communicating the problem to <code>PyDEns</code></strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/pydens/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>